smoke-test/src/test/resources/topology/graphml/test-topology.xml

```diff
@@ -28,6 +30,29 @@
         <node id="east">
             <data key="label">East Region</data>
         </node>
+        <node id="center">
```

Comment:
Is it possible to add the `foreignID` and `foreignSource` to an already existing node, instead of creating new nodes?